modules/tutorials/pages/schema-design-guide.adoc

```diff
@@ -1 +1,267 @@
+
+The following schema creates the necessary time vertices and their relationships:
+
+[source.wrap,gsql]
+----
+// 1. Define Vertex Types:
+CREATE VERTEX Year (PRIMARY_ID year_val INT);
+CREATE VERTEX Month (PRIMARY_ID month_val INT, month_name STRING);
+CREATE VERTEX Day (PRIMARY_ID day_val INT);
+CREATE VERTEX Product (PRIMARY_ID product_id STRING);
+
+// 2. Define Edge Types to establish hierarchy and relationships:
+// Connects a Year to its Months
+CREATE DIRECTED EDGE HAS_MONTH (FROM Year, TO Month);
+
+// Connects a Month to its Days
+CREATE DIRECTED EDGE HAS_DAY (FROM Month, TO Day);
+
+// Connects a Product to a specific Day (where an event related to the product occurred)
+CREATE DIRECTED EDGE PRODUCED_ON (FROM Product, TO Day);
+
+----
+
+=== Focal Hub
+
+A *focal hub* is a *central vertex* that connects to many other vertices via multiple edges. It can become a performance bottleneck in your schema.
+
+==== Managing Hub Nodes
+
+If you need to use hub nodes, make sure to optimize them using secondary indexes.
+
+=== Use Case-Based Schema Design
+
+Graph schema design should always reflect your use case and query requirements. The type of relationships you need to track will guide your choice of schema design.
+
+==== Event-Centered Schema
+
+This design focuses on events as the central element of your application. It’s ideal for use cases where you need to analyze event-driven data.
+For example, in an e-commerce platform, events like purchases, returns, or reviews could be central to understanding the relationships between customers and products.
+
+For example, in an e-commerce schema, *purchases* and *product reviews* would be modeled as key events, with relationships showing how customers interact with products.
+
+image::Event-Centered Schema.png[]
+
+==== User-Centered Schema
+
+This design focuses on user behavior and interactions, making it ideal for applications like social networks or user analytics.
+In a user-centered schema, you would model users and their relationships with products, services, or other users.
+
+For example, in a social network schema, you could model *user interactions* (such as *comments, likes, and shares*) and track user behavior over time.
+
+image::User-Centered Schema.png[]
+
+[TIP]
+====
+* *Choose an event-centered schema* when your primary focus is on analyzing *events* (like *purchases or transactions*).
+* *Choose a user-centered schema* when you want to track *user behavior* and *interactions* across your system.
+====
+
+=== Recommendation
+
+Here are some key recommendations for schema design:
+
+* *Edge Types:* Use multiple edge types when specificity is important; for simpler designs, use a single edge type with attributes.
+
+* *Attributes vs. Vertices:* Model frequently queried properties as vertices and use attributes for less critical data.
+
+* *Events:* For event-based analytics, model events as vertices; use edge attributes for simpler scenarios.
```

Comment:
Not sure what point you are trying to make with "use edge attributes for simpler scenarios."